vm/actor/tests/common/mod.rs

```diff
@@ -229,14 +242,28 @@ impl<'a, BS: BlockStore> MockRuntime<'a, BS> {
             value,
             send_return,
             exit_code,
-        })
+        });
+        ()
+    }
+
+    #[allow(dead_code)]
+    pub fn expect_create_actor(&mut self, code_id: Cid, address: Address) {
+        let a = ExpectCreateActor { code_id, address };
+        self.expect_create_actor = Some(a);
+    }
+
+    #[allow(dead_code)]
+    pub fn set_caller(&mut self, code_id: Cid, address: Address) {
+        self.caller = address.clone();
+        self.caller_type = code_id.clone();
+        self.actor_code_cids.insert(address, code_id);
     }
 }
 
 impl<BS: BlockStore> Runtime<BS> for MockRuntime<'_, BS> {
+    // Cuasing a test to fail have to implement to pass. Ask about the
```

Comment:
commenting to reference above as it's related, this impl is fine, but probably want to add back the `self.require_in_call()`

Reply:
Yeah, please add that back